packages/rocketchat-ui-admin/client/users/adminInviteUser.js

```diff
@@ -1,7 +1,8 @@
 import toastr from 'toastr';
 Template.adminInviteUser.helpers({
-	isAdmin() {
-		return RocketChat.authz.hasRole(Meteor.userId(), 'admin');
+	isAuthorized() {
+		return 	RocketChat.authz.hasRole(Meteor.userId(), 'admin') ||
+				RocketChat.authz.hasAtLeastOnePermission('bulk-register-user');
```

Comment:
This allows users who are not admins but have the proper permissions to invite users